package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "account-lookup-service",
   "description": "Account Lookup Service is used to validate Party and Participant lookups",
-  "version": "8.2.1",
+  "version": "8.2.3",
```

Comment:
Needed an extra bump because CircleCI was restoring an invalid cache for `8.2.2`